salt/matchers/pillar_match.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+'''
+This is the default pillar matcher function.
+
+NOTE: These functions are converted to methods on the Matcher class during master and minion startup.
+This is why they all take `self` but are not defined inside a `class:` declaration.
+'''
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+from salt.ext import six  # pylint: disable=3rd-party-module-not-gated
+from salt.defaults import DEFAULT_TARGET_DELIM  # pylint: disable=3rd-party-module-not-gated
+import salt.utils.data  # pylint: disable=3rd-party-module-not-gated
+
+log = logging.getLogger(__name__)
+
+
+def match(self, tgt, delimiter=DEFAULT_TARGET_DELIM):
+    '''
+    Reads in the pillar glob match
+    '''
+    log.debug('pillar target: %s', tgt)
+    if delimiter not in tgt:
+        log.error('Got insufficient arguments for pillar match '
+                  'statement from master')
```

Comment:
Same here. OK there are more. Could you please look over all those messages and make a better feedback? Because `log.error` is read by admins, that might have absolutely no clue of Salt's internals (and usually they don't).

Reply:
Likewise